src/algorithms/reco/ElectronReconstructionConfig.h

```diff
@@ -6,8 +6,8 @@ namespace eicrecon {
 
 struct ElectronReconstructionConfig {
 
-    double min_energy_over_momentum = 0.9;
-    double max_energy_over_momentum = 1.2;
+    double min_energy_over_momentum = 0.7;
+    double max_energy_over_momentum = 1.3;
```

Comment:
Revert back, as this applys to the `ReconstructedElectrons` collection which has broader use than a DIS electron finder. The stricter cut can be applied to a new collection (`ReconstructedElectronsForDIS`):
```suggestion
    double min_energy_over_momentum = 0.9;
    double max_energy_over_momentum = 1.2;
```